rif/relay/versions.md

```diff
@@ -38,5 +38,29 @@ It is important to recall that - as a security measure - version 1 contracts dep
     * The maximum gas estimation in the RIF Relay Server is more precise now.
     * A new utility function is available to estimate the maximum gas a relay transaction would consume, based in a linear fit estimation. This can be used in applications that don't want to sign a payload each time they need an approximation of the cost of relaying the transaction.
 * Paymaster verifications are done off-chain to optimize gas costs, thus the paymasters are now called Verifiers and they are not part of the on-chain relay flow nor do they handle payments at all.
-* Big gas cost optimization.
-* Security issues fixed.
\ No newline at end of file
+* Gas cost optimization.
+* Security issues.
+
+
+## Version 1
+
+### Overview
+
+Including a revenue-sharing mechanism to the RIF Relay service should not introduce a price penalty to the RIF Relay users. With this restriction in mind, we proposed to modify the address used to pay (in tokens) for a successful transaction relay (or deploy). 
+
+In its current implementation, the RelayRequest and the DeployRequest include a relayWorker attribute to identify which account paid for the gas. The RIF Relay SmartWallet pays directly to this account the number of tokens negotiated for the Relay (or Deploy) service. The relayWorker attribute can be replaced with the address of a Smart Contract containing the revenue sharing logic. 
```

Comment:
```suggestion
In its current implementation, we removed the relayWorker attribute from the RelayRequest and DeployRequest to include a `feesReceiver` attribute in the RelayData. The RIF Relay SmartWallet pays directly to this account the number of tokens negotiated for the Relay (or Deploy) service. The relayWorker attribute can be replaced with the address of a Smart Contract containing the revenue sharing logic.
```

Reply:
We need to discuss this, since I think that I think in a different approach

Reply:
We agreed on specifying the implementation we are referring to just to clarify that when we say in "in its current implementation" we refer to version 0.2

Reply:
updated